index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <title>Bowling Scorecard</title>
+    <link rel='stylesheet' href='css/stylesheet.css'>
+  </head>
+
+  <body>
+
+    <section>
+      <h1> Bowling Scorecard <span id='total-score'></span>
+      <h2><span id='scorecard'></span></h2>
+      
+      <p>
+        <p>Frame:<span id='frame-number'></span>
+        </p>
+        <button id='begin-frame'>Begin Frame</button>
+        <p>
+        <form id='enter-points' action= 'index.html'>
+          <input type='text' id='throw-input' placeholder="Enter Points"></input>
+          <input type='submit '></input>
+        </form>
+        </p>
+      </p>
+      <img src='https://bloximages.chicago2.vip.townnews.com/cumberlink.com/content/tncms/assets/v3/editorial/3/26/32657b72-9d18-5a01-a161-3fa67be1a4ba/59231a809f884.image.jpg?resize=1200%2C821' width = 800px height = 400px>
+    </section>
```

Comment:
why did you choose to link the image from the web instead of including in your project as static asset?